src/frontend/src/pages/ChangeRequestDetailPage/ChangeRequestDetailsView.tsx

```diff
@@ -105,27 +133,103 @@ const ChangeRequestDetailsView: React.FC<ChangeRequestDetailsProps> = ({
     setAnchorEl(null);
   };
 
-  const unreviewedActionsDropdown = (
-    <div>
-      <NERButton
-        endIcon={<ArrowDropDownIcon style={{ fontSize: 28 }} />}
-        variant="contained"
-        id="unreviewed-cr-actions-dropdown"
-        onClick={handleClick}
-      >
-        Actions
-      </NERButton>
-      <Menu open={dropdownOpen} anchorEl={anchorEl} onClose={handleDropdownClose}>
-        <MenuItem onClick={handleReviewOpen} disabled={!isUserAllowedToReview}>
-          Review
-        </MenuItem>
-        <Divider />
-        <MenuItem disabled={!isUserAllowedToDelete} onClick={handleDeleteOpen}>
-          Delete
-        </MenuItem>
-      </Menu>
-    </div>
-  );
+  const handleRequestReviewerClick = async () => {
+    if (reviewerIds.length === 0) return;
+    try {
+      await requestCRReview.mutateAsync({ userIds: reviewerIds });
+    } catch (e) {
+      if (e instanceof Error) {
+        toast.error(e.message);
+      }
+    }
+  };
+
+  const isRequestAllowed = changeRequest.submitter.userId === currentUser.userId;
+
+  const unreviewedActionsDropdown = () => {
+    if (isRequestAllowed && changeRequest.status === ChangeRequestStatus.Open) {
```

Comment:
you can abstract these two component you return into helper component, then convert this to a ternary: `isRequestAllowed && changeRequest.status === ChangeRequestStatus.Open ? <OneComponent/> : <SecondComponent />